src/plots/cartesian/axis_defaults.js

```diff
@@ -75,6 +75,12 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
     coerce('range');
     containerOut.cleanRange();
 
+    coerce2('showspikes');
```

Comment:
The short answer is "because it made things work". The slightly longer answer is that with just coerce(), it seems like some of the gl charts wind up including the cartesian axis values, but because the various spike options aren't defined in the gl3d layout options, things exploded pretty badly.

Some things seem to alternately get defined as axes options or layout options, and I saw that coerce2 was defined above to always pass in layout options (which is where these options are defined for the non-gl charts), so it meant that even when this got included by the gl3d charts, it didn't cause issues.

Here's the stack you get on a gl3d chart when this uses `coerce`:
```
Uncaught TypeError: Cannot read property 'dflt' of undefined
    at Object.exports.coerce (coerce.js:242)
    at coerce (axis_defaults.js:30)
    at handleAxisDefaults (axis_defaults.js:81)
    at supplyLayoutDefaults (axis_defaults.js:44)
    at handleGl3dDefaults (defaults.js:92)
    at handleSubplotDefaults (subplot_defaults.js:71)
    at Object.supplyLayoutDefaults (defaults.js:32)
    at Object.plots.supplyLayoutModuleDefaults (plots.js:1050)
    at Object.plots.supplyDefaults (plots.js:475)
    at Object.Plotly.plot (plot_api.js:136)
```